source/db/ar-projects.json

```diff
@@ -5097,11 +5097,11 @@
   {
     "development_stage": "released",
     "description": "توزيعة  أندرويد حرة  بالكامل.",
-    "license_url": "http://replicant.us/about/",
+    "license_url": "https://git.replicant.us/replicant/manifest/blob/replicant-6.0-dev/README.mkdn",
```

Comment:
Yes, pretty much the same. It's beyond the scope of the project to keep a list of all subprojects and their respective licenses. The individual repos need to be checked for their licenses.